packages/v4/patternfly-docs/content/design-guidelines/components/popover/popover.md

```diff
@@ -49,14 +32,32 @@ Use link button styling for the linked question, and add a blue question-circle
  
 To provide additional information about an alert, use an alert popover. The alert popover will open upon hover or click.
  
-## Tooltips versus popovers
+### When to use tooltips vs. popovers
 Both [tooltips](/components/tooltip/design-guidelines) and popovers provide more information in context for users. However, they’re different in a few ways:
  
 - Tooltips are used for identification purposes, while popovers are used for added description or information in context. 
 - Tooltips contain short descriptions or labels, while popovers contain longer descriptions, formatted text, and optional images or links.
 - Tooltips appear on hover, while popovers appear on click.  
  
-## Content
+For information on other forms of on-screen help, see [Tooltip](/components/tooltip/design-guidelines) and [Hint](/components/hint/design-guidelines).
+ 
+## Variations
+ 
+**Default popovers** are plain-text and do not include icons. They are used to convey additional, non-imperative information to the user.
```

Comment:
Wonder if we should highlight these more by making it H3s with a description underneath. For example:

[h3] Default popovers 
These popovers are plain-text and do not include icons. They are used to convey additional, non-imperative information to the user.

[h3] Alert popovers
These popovers are similar to ...e tc